### DISCUSSION TITLE
BUG ImportError

### DISCUSSION THREAD

**Discussion by NMThanh123** [PaddlePaddle/PaddleOCR#12895] [Q&A]
Cannot import name 'MSRA' from 'paddle.fluid.initializer' (/usr/local/lib/python3.10/dist-packages/paddle/fluid/initializer.py)
I use versions:
paddlepaddle-gpu: 2.5.0
paddleslim: 2.3.2
I want to export quantized model but I get error. Please help me if you have suggestion

**Comment by SWHL** [comment]
This is error of Paddle, please go to [Paddle](https://github.com/PaddlePaddle/Paddle) repo for question.
